config/custom-environment-variables.yaml

```diff
@@ -14,9 +14,9 @@ services:
     scope: 'openid profile roles manage-user search-user'
     responseType: 'code'
 session:
-  secret: 'local'
```

Comment:
THis config setting is removed here but think it is still used in the 'property-volume' index.ts and 'session' index.js